src/qml/editorwidgets/Range.qml

```diff
@@ -51,9 +51,6 @@ EditorWidgetBase {
     }
 
     onTextChanged: {
-      if ( value != undefined && value != text )
-        value = text
-
```

Comment:
```
      if ( value != text )
        value = text
```
Is this still causing an issue? I would like to keep the warning away if possible. 

If not, it was not crucial, QML detects and prevents circular dependencies itself.

Reply:
I can't see a warning.